less/theme.less

```diff
@@ -196,96 +192,15 @@ body {
     background-color: @body-bg;
 }
 
-code, pre, kbd {
-    font-family: @font-family-monospace;
-}
-
-//
-// Buttons
-// --------------------------------------------------
-
-// Mixin for generating btn styles
-.btn-styles(@btn-color; @btn-border; @btn-text) {
-
-    display: inline-block;
-    padding: 0.5rem 1rem;
-    font-family: @font-family-btn;
-
-    border: solid 1px @btn-border;
-    border-radius: 3px;
-
-    text-align: center;
-    text-decoration: none;
-
-    background-color: @btn-color;
-    color: @btn-text;
-    cursor: pointer;
-    min-width: 130px;
-
-    &:hover,
-    &:focus,
-    &:active,
-    &.active {
-        background-color: @btn-color;
-        border-color: darken(@btn-border, 12%);
-        color: @btn-text;
-        text-decoration: none;
+.browser-ie11({
+    // fix egregious vertical centering issue with Lato in IE11 by falling back to sans-serif
+    body {
+        font-family: sans-serif !important;
```

Comment:
ehhh... you could maybe just make this more specific, if there was a class on `body`, e.g. `body.ie11.ie11` or whatever